core/src/syn/parser/stmt/define.rs

```diff
@@ -31,7 +31,9 @@ use crate::{
 impl Parser<'_> {
 	pub async fn parse_define_stmt(&mut self, ctx: &mut Stk) -> ParseResult<DefineStatement> {
 		match self.next().kind {
-			t!("NAMESPACE") => self.parse_define_namespace().map(DefineStatement::Namespace),
+			t!("NAMESPACE") | t!("ns") => {
+				self.parse_define_namespace().map(DefineStatement::Namespace)
+			}
 			t!("DATABASE") => self.parse_define_database().map(DefineStatement::Database),
```

Comment:
Should `DATABASE` also be aliased as `DB`?

Reply:
`DATABASE` already does that. 'DATABASE', 'DB' and all its casing variants result in the `Keyword::Database` token. `NAMESPACE` also aliases to `NS` and `Ns` and `nS`. However with the introduction of token gluing I had to split up tokens and introduce the `ns` token for nanosecond suffixes in durations, this token overlaps with the namespace token so I have to add it here.

Reply:
The bug here only happened when you wrote `USE ns test;`. `USE NS test;` `USE Ns test;` etc, did work.

Reply:
That makes total sense. Thank you for the context :+1: